tests/foreman/ui_airgun/test_contenthost.py

```diff
@@ -515,3 +554,250 @@ def test_positive_virt_who_hypervisor_subscription_status(session):
+        )
+        assert result['overview']['hosts_table'][0]['Status'] == 'success'
+
+
+def test_install_modular_errata(session, vm_module_streams):
+    """Populate, Search and Install Modular Errata generated from module streams.
+
+    :id: 3b745562-7f97-4b58-98ec-844685f5c754
+
+    :expectedresults: Modular Errata should get installed on content host.
+
+    :CaseLevel: System
+
+    :CaseImportance: High
+    """
+    with session:
+        stream_version = "0"
+        run_remote_command_on_content_host(
+            'dnf -y upload-profile',
+            vm_module_streams
+        )
+        result = session.contenthost.execute_module_stream_action(
+            vm_module_streams.hostname,
+            action_type='Install',
+            module_name="kangaroo",
+            stream_version=stream_version,
+        )
+        assert result['overview']['hosts_table'][0]['Status'] == 'success'
+
+        # Downgrade rpm package to generate errata.
+        run_remote_command_on_content_host(
+            'dnf downgrade kangaroo -y',
+            vm_module_streams
+        )
+        module_stream = session.contenthost.search_module_stream(
+            vm_module_streams.hostname,
+            "kangaroo",
+            status='Upgrade Available'
+        )
+        assert module_stream[0]['Name'] == "kangaroo"
+
+        # Verify the errata
+        chost = session.contenthost.read(vm_module_streams.hostname, 'errata')
+        assert FAKE_0_MODULAR_ERRATA_ID in {errata['Id'] for errata in chost['errata']['table']}
+
+        # Install errata
+        result = session.contenthost.install_errata(
+            vm_module_streams.hostname,
+            FAKE_0_MODULAR_ERRATA_ID,
+            install_via='via remote execution'
+        )
+        assert result['overview']['hosts_table'][0]['Status'] == 'success'
+
+        # Ensure errata installed
+        assert not session.contenthost.search_module_stream(
+            vm_module_streams.hostname,
+            "kangaroo",
+            status='Upgrade Available'
+        )
+
+
+@tier3
+def test_module_status_update_from_content_host_to_satellite(session, vm_module_streams):
+    """ Verify dnf upload-profile updates the module stream status to Satellite.
+
+    :id: d05042e3-1996-4293-bb01-a2a0cc5b3b91
+
+    :expectedresults: module stream status should get updated in Satellite
+
+    :CaseLevel: System
+
+    :CaseImportance: High
+    """
+    with session:
+        stream_version = "0.71"
+        profile = "flipper"
+        run_remote_command_on_content_host(
+            'dnf -y upload-profile',
+            vm_module_streams
+        )
+
+        # reset walrus module streams
+        run_remote_command_on_content_host(
+            'dnf module reset walrus -y',
+            vm_module_streams
+        )
+
+        # install walrus module stream with flipper profile
+        run_remote_command_on_content_host(
+            'dnf module install walrus:0.71/flipper -y',
```

Comment:
You could use variables defined earlier, like:
```
'dnf module install {}:{}/{} -y'.format(module_stream, stream_version, profile)
```
(for this to work, you need to define module_stream first)